IDR/notebook/scripts/nbconverted/1.describe_screens.py

```diff
@@ -59,22 +59,32 @@ def extract_study_info(session, screen_id):
     return details_
 
 
-def describe_screen(screen_id):
+def describe_screen(screen_id, sample, imaging_method, study_name):
```

Comment:
In the future, you'll likely want to abstract this function into its own `describe_utils.py` (or something named appropriately) script for import

Reply:
By abstract do you mean simply modularize it or make it completely generalizable?

Reply:
I mean that you might consider [refactoring](https://bids.berkeley.edu/news/joy-code-refactoring) this function into a separate file (e.g. called `describe_utils.py`) that you can import back here. Doing so would provide [abstraction](https://en.wikipedia.org/wiki/Abstraction_(computer_science)) so that a reader can focus on the core functionality of your notebook.

Reply:
@gwaybio Okay perfect that was my understanding too.